templates/default/partials/version/download.html.twig

```diff
@@ -36,26 +36,52 @@
                                 <p><a href="https://ddev.readthedocs.io/en/stable/" title="DDEV Documentation" target="_blank" rel="noopener">DDEV</a> simplifies integrating the power and consistency of containerization into your workflows. Set up environments in minutes; switch contexts and projects quickly and easily; speed your time to deployment. DDEV handles the complexity. You get on with the valuable part of your job.</p>
                                 <p><b>Before proceeding, make sure your DDEV installation is up to date.</b></p>
                                 <p>In a new project folder using your favorite shell, run these lines:</p>
-                                <pre><code>ddev config --project-type=typo3 --docroot=public --create-docroot{{ php_version_option }}
+                                <pre><code>ddev config --project-type=typo3 --docroot=public{{ php_version_option }}
 ddev composer create --no-install "typo3/cms-base-distribution:^{{ package_version }}"
 ddev composer install
 {% if version == 12 %}
 ddev restart
 {% endif %}
 {% if console %}
+{% if version == 10 %}
```

Comment:
Version 10 is in ELTS and therefore the whole section is not rendered anymore. I think it's safe to remove all special v10 handling.